core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -152,7 +155,7 @@ oppia.controller('FeedbackTab', [
           currentContent: function() {
             var stateName;
             if (constants.USE_NEW_SUGGESTION_FRAMEWORK) {
-              stateName = $scope.activeThread.change_cmd.state_name;
+              stateName = $scope.activeThread.suggestion.change_cmd.state_name;
```

Comment:
You should have a domain object for Suggestion in the frontend. This can be introduced in the next PR but I would recommend doing it immediately after this one. Please make sure to file an issue for it if you aren't handling it here.

Accessing "backend properties" like change_cmd directly is a no-no. @aks681 can give you advice on how to handle the backend-to-frontend transition and domain objects in general.

Reply:
I am working on this now. I think it would be better to do it in this PR itself.

Reply:
Done.